src/components/board.js

```diff
@@ -2,34 +2,68 @@ import React from 'react';
 import Card from './Card/card';
 class Board extends React.Component {
     constructor(props) {
-        super(props);
+        super(props)
+        this.change = this.change.bind(this);
         this.state = {
             store: [],
+            flipped: [],
         };
 
     }
+    change(i) {
+        console.log("parent   "+i)
+        const newflipped = this.state.flipped.slice();
+        if (newflipped[i] == false) {
+            newflipped[i] = !newflipped[i];
+            this.setState({ 
+              flipped: newflipped,
+            });
+        }
+        
+        console.log(this.state.flipped)
 
+    }
     start(size) {
+        var k = 0;
         var store = [];
+        var flipped= Array(size*size).fill(false);
```

Comment:
I can't make proper sense of some variables, such as `store`,  `flipped`. Need to provide proper variable names to signify what it means.